### DISCUSSION TITLE
Possible to show Filename after playing next file for a few seconds? OSD is able to be toggled but stays until disabled, any way to have it toggled on and off for 3 seconds after playing next file?

### DISCUSSION THREAD

**Discussion by arukom** [smplayer-dev/smplayer#913] [Q&A]
Hi, 

i am on linux mint and am using:
Version: 21.10.0 (revision 10000)
Using Qt 5.15.3 (compiled with Qt 5.15.2)
Using MPV 0.34.1

is there a way to have the OSD show the title of the file for a brief second after opening new file or playing next file as to know if i am watching a series or a tv show  which episode number i am on?

Ty

**Comment by arukom** [comment]
oh i was able to do it with `--osd-playing-msg="${filename}"`
to add in advanced -> mplayer/mpv -> options

i am unable to show it for longer than 2 second tho which is a bummer but will do for now..

**Reply by Raza-0007** [reply]
In addition to this: `-osd-playing-msg=file: ${filename}`

Pass this to mpv: `-osd-playing-msg-duration=5000`

This will display the filename on OSD for 5 seconds. Time is in "ms" so you can change it to whatever you prefer.

~Raza

**Reply by arukom** [reply]
since i am using linux mint, this is sadly not an option i am guessing this is for a newer release i am using an outdated one. The newer flatpak version works with `-osd-playing-msg-duration=5000` but not the version i have.

anyway, i changed it up again in the smplayer.ini. located in /.config/smplayer
"osd_show_filename_duration=4000" then on smplayer settings preferences -> advanced -> advanced -> in the "run following commands everytime a file is opened i wrote in "show_filename_osd".
now it work for 4 seconds

thank you!
